### PR TITLE
Changes to get ie 11 to work, pull in polyfills for fetch and promise.

### PR DESCRIPTION
**Pull request by alogan519**
Use d2l-t* elements instead of table elements.

### CODE REVIEW

File: d2l-rubric-criteria-group.html [hunk 1]
```diff
@@ -35,48 +40,54 @@
 			}
 
 		</style>
-		<d2l-table-wrapper>
-			<table class="d2l-table" summary="[[localize('rubric_summary_a11y')]]">
-				<thead>
-					<tr>
-						<th>
+		<style type="text/css">
+			.btn {
```
Comment (djwester) [inline d2l-rubric-criteria-group.html]: What is this button styling for?

File: d2l-rubric-criteria-group.html [hunk 2]
```diff
@@ -35,48 +40,54 @@
...
+				border-radius: 0;
+			}
+			table.table tbody tr {
+				transition: initial;
+			}
+		</style>
+		<d2l-table summary="[[localize('rubric_summary_a11y')]]">
```
Comment (awikkerink) [inline d2l-rubric-criteria-group.html]: Summary won't be read by screen readers. Maybe use `aria-describedby`?

File: d2l-rubric-criteria-group.html [hunk 3]
```diff
@@ -35,48 +40,54 @@
 			}
 
 		</style>
-		<d2l-table-wrapper>
-			<table class="d2l-table" summary="[[localize('rubric_summary_a11y')]]">
-				<thead>
-					<tr>
-						<th>
+		<style type="text/css">
```
Comment (jstefaniuk-d2l) [inline d2l-rubric-criteria-group.html]: Do you need to have 2 `<style>` tags? Or can it all be moved to one?

File: d2l-rubric-criteria-group.html (hunk 1 above)
Comment (alogan519) [reply d2l-rubric-criteria-group.html]: Thanks for catching this, forgot I was playing around with this.

File: d2l-rubric-criteria-group.html (hunk 2 above)
Comment (djwester) [reply d2l-rubric-criteria-group.html]: Current rubric table uses summary, which is why we did this here. Aria is probably a more standard thing though, so probably worth changing

File: d2l-rubric.html [hunk 4]
```diff
@@ -5,6 +5,9 @@
 <link rel="import" href="../d2l-hm-constants-behavior/d2l-hm-constants-behavior.html">
 <link rel="import" href="../d2l-typography/d2l-typography-shared-styles.html">
 <link rel="import" href="./localize-behavior.html">
+<link rel="import" href="../promise-polyfill/promise-polyfill.html">
+
+<script src="../fetch/fetch.js"></script>
```
Comment (dbatiste) [inline d2l-rubric.html]: We should be importing `d2l-fetch`, yes?

Comment (jstefaniuk-d2l) [reply d2l-rubric.html]: I may be wrong but I don't think d2l-fetch includes the polyfill

Comment (ryantmer) [reply d2l-rubric.html]: Oh, okay, spent another 14 seconds looking. This repo is already using `d2l-fetch` correctly, TIL. Generally I'd say it's up to the thing _using_ `d2l-rubric` (e.g. BSI) to include the polyfills, but maybe there's a case where that's not sufficient here?

Comment (alogan519) [reply d2l-rubric.html]: So one advantage of doing the polyfill here was that the polymer demo works on ie 11 now.

Comment (jstefaniuk-d2l) [reply d2l-rubric.html]: You may just want to put the import on the demo page in that case

Comment (ryantmer) [reply d2l-rubric.html]: Yeah, I'd suggest what @jstefaniuk-d2l said, as well as moving it to the `devDependencies` in that case :)

Comment (alogan519) [reply d2l-rubric.html]: ok, sounds good.

File: bower.json [hunk 5]
```diff
@@ -33,11 +33,14 @@
     "d2l-localize-behavior": "^1.1.0",
     "d2l-typography": "^6.1.2",
     "polymer": "1 - 2",
-    "iron-resizable-behavior": "^2.1.0"
+    "iron-resizable-behavior": "^2.1.0",
+    "d2l-offscreen": "^3.0.3"
```
Comment (dbatiste) [inline bower.json]: Nitpick: our general practice is to keep the deps in alpha order.  Just helps when searching through things sometimes.

Comment (awikkerink) [reply bower.json]: Bower doesn't enforce alpha order when running `bower install`. It kinda sucks

Comment (dbatiste) [reply bower.json]: Yeah, it's a minor manual adjustment when including dependencies.

**Comment by awikkerink** [comment]
Maybe `role=grid` could be used on d2l-table to fix the aria errors?

**Comment by awikkerink** [comment]
I think grid may be a more appropriate role anyway.

https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/js/Constants.js doesn't define table as a valid role.

**Comment by alogan519** [comment]
Yup switching to role grid fixes it, Anthony should I fix that here or make a change to d2l-table?

File: bower.json (hunk 5 above)
Comment (alogan519) [reply bower.json]: yup, I can reorder manually.

File: test/components/d2l-rubric-criteria-group.html [hunk 6]
```diff
@@ -24,7 +24,7 @@
 			</template>
 		</test-fixture>
 		<script>
-			a11ySuite('basic-criteria');
-		  </script>
+			a11ySuite('basic-criteria', ['badAriaRole', 'ariaRoleNotScoped']);
```
Comment (awikkerink) [inline test/components/d2l-rubric-criteria-group.html]: Maybe put a `FIXME` here to note why these are disabled?

**Pull request merged**